routers/web/user/home.go

```diff
@@ -887,7 +767,9 @@ func getUserIssueStats(ctx *context.Context, filterMode int, opts *issue_indexer
 	{
 		openClosedOpts := opts.Copy()
 		switch filterMode {
-		case issues_model.FilterModeAll, issues_model.FilterModeYourRepositories:
+		case issues_model.FilterModeAll:
```

Comment:
Either remove this line or add a comment like `// no-op` ?

Reply:
Updated: 1de6c7cc27941beaf1773b1e3167ab28168565af